code/lab1-tdd-arabic-roman-converter/src/test/java/ru/unn/agile/ArabicRomanConverter/ArabicToRomanTest.java

```diff
@@ -0,0 +1,157 @@
+package ru.unn.agile.ArabicRomanConverter;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ArabicToRomanTest {
+
+    private String convertToRoman(final int arabic) {
+        ArabicToRomanConverter converter = new ArabicToRomanConverter();
+        return converter.convert(arabic);
+    }
+
+    @Test
+    public void canConvertToEmptyString() {
+        String romanNumber = convertToRoman(0);
```

Comment:
Why convertToRoman(0) should return ""?
Maybe in that case you should throw IllegalArgumentException with message "Roman number doesn't exist for Arabic {}"

Reply:
Done